### PR TITLE
FS-1173 - Fix ssl timeout

### PR DESCRIPTION
**Pull request by harryyo**
### Fix SSL Timeout
The implementation of Talisman on this repo forced redirection of http traffic to https - however this application is currently designed to be deployed on an .apps.internal hosting space within GovUK PaaS.
apps.internal hosts use http only (not https) and are routed by GovUK PaaS to be inaccessible from the public internet - as result **this PR removes Talisman and forcing of http traffic to https** to allow the deployment to be http-only, delegating request security to GovPaaS routing.

### CODE REVIEW

File: manifest.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 applications:
 - name: funding-service-design-notification-dev
-  memory: 64M
+  memory: 256M
```
Comment (gidsg) [inline manifest.yml:4]: Why are we bumping the memory in this PR? Can we revert this file if it is unrelated?

Comment (Adam-W1) [reply manifest.yml:4]: Maybe a separate PR for this one to keep SOC?

**Review by Adam-W1 — APPROVED** [review]
looks good - the description is really helpful 👍 (there is a point from @gidsg about removing the memory bump which may be better on another pr for SeperationOfConcern)
